docker/build_artifacts/sagemaker/python_service.py

```diff
@@ -26,8 +27,8 @@
 import tfs_utils
 
 SAGEMAKER_MULTI_MODEL_ENABLED = os.environ.get("SAGEMAKER_MULTI_MODEL", "false").lower() == "true"
-MODEL_DIR = "models" if SAGEMAKER_MULTI_MODEL_ENABLED else "model"
-INFERENCE_SCRIPT_PATH = f"/opt/ml/{MODEL_DIR}/code/inference.py"
+MODEL_DIR = "" if SAGEMAKER_MULTI_MODEL_ENABLED else "model/"
```

Comment:
Why needs to change the dir structures?

Reply:
The problem with this path is that `/opt/ml/models` is a read-only file system for multi-model endpoints. Assuming the structure as shown below in an s3 bucket, the universal `inference.py` can’t be downloaded to `/opt/ml/models/code/inference.py.` 

![image](https://user-images.githubusercontent.com/110572198/210860589-aebc9fad-977f-49eb-9e79-c903e64b4569.png)

This problem has been highlighted in other Github issues (https://github.com/aws/sagemaker-tensorflow-serving-container/issues/212 and https://github.com/aws/sagemaker-tensorflow-serving-container/issues/211) as well and a PR (https://github.com/aws/sagemaker-tensorflow-serving-container/pull/215) was created to solve the issue but it was not merged.

We can use the path `/opt/ml/code/inference.py` for the universal `inference.py` file in the case of a multi-model endpoint.